test/integration/cleanliness_test.rb

```diff
@@ -11,14 +11,4 @@
     # if it is not avoidable see http://stackoverflow.com/questions/7163264
     File.read('Gemfile.lock').wont_include 'rails-assets-bootstrap '
   end
-
-  if ActiveRecord::Base.connection.adapter_name == "Mysql2"
-    it "uses the right row format in mysql" do
```

Comment:
I don't really understand what it's testing, the travis config? People can run samson without utf8mb4, so why do we care? I'd argue the monkey patch is also unnecessary and should be optionally included by users.